libs/libc/stream/lib_stdsostream.c

```diff
@@ -70,18 +71,19 @@ static void stdsostream_putc(FAR struct lib_sostream_s *self, int ch)
 static int stdsostream_puts(FAR struct lib_sostream_s *self,
                             FAR const void *buffer, int len)
 {
-  FAR struct lib_stdsostream_s *stdsostream = (FAR struct lib_stdsostream_s *)self;
+  FAR struct lib_stdsostream_s *ostream =
```

Comment:
why change

Reply:
nxstyle complained that the line was too long so I shortened the names to be under 80 characters

Reply:
I'm just now realizing that it isn't required for the `(` to be at the same indent level as the `=` so I could have kept the same name and pushed the second line to the left more:

```
FAR struct lib_stdsostream_s *stdsostream = 
                                                        (FAR struct lib_stdsostream_s *)self;
```